app/Jobs/FetchNftActivity.php

```diff
@@ -42,9 +43,19 @@ public function uniqueId(): string
      */
     public function handle(): void
     {
+        Log::info('FetchNftActivity Job: Processing', [
+            'token_number' => $this->nft->token_number,
```

Comment:
best to include the collection id or name as well, otherwise these token numbers will be ambiguous between collections